.devcontainer.json

```diff
@@ -9,46 +9,45 @@
         "8123": {
             "label": "Home Assistant",
             "onAutoForward": "notify"
-        },
-        "0-8122": {
-            "label": "Auto-Forwarded - Other",
-            "onAutoForward": "ignore"
-        },
-        "8124-999999": {
-            "label": "Auto-Forwarded - Other",
-            "onAutoForward": "ignore"
         }
     },
     "customizations": {
         "vscode": {
             "extensions": [
-                "ms-python.python",
+                "charliermarsh.ruff",
                 "github.vscode-pull-request-github",
-                "ryanluker.vscode-coverage-gutters",
+                "ms-python.python",
                 "ms-python.vscode-pylance",
-                "visualstudioexptteam.vscodeintellicode",
-                "redhat.vscode-yaml",
-                "esbenp.prettier-vscode"
+                "ryanluker.vscode-coverage-gutters",
+                "visualstudioexptteam.vscodeintellicode"
             ],
             "settings": {
-                "files.eol": "\n",
-                "editor.tabSize": 4,
-                "python.defaultInterpreterPath": "/usr/local/bin/python",
-                "python.pythonPath": "/usr/local/python/bin/python",
-                "python.analysis.autoSearchPaths": false,
-                "python.linting.pylintEnabled": true,
-                "python.linting.enabled": true,
-                "python.formatting.provider": "black",
-                "python.formatting.blackPath": "/usr/local/py-utils/bin/black",
+                "[python]": {
+                    "editor.defaultFormatter": "charliermarsh.ruff"
+                },
                 "editor.formatOnPaste": false,
-                "editor.formatOnSave": false,
+                "editor.formatOnSave": true,
```

Comment:
Is this a HA-standard we need to comply with? As previously mentioned, I'm really not a fan of the formatOnSave behaviour, and the validate-workflows should still keep format errors from being merged into master, right?